File: app/src/main/java/com/ftiuksw/appkantinfti/LoginActivity.java
package com.ftiuksw.appkantinfti;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.login);
    }

    public void go2Home(View view) {
        Intent intent = new Intent(this, HomeActivity.class);
        EditText editText = (EditText) findViewById(R.id.FirstName_ID);
        String message = editText.getText().toString();
        intent.putExtra("Username", message);
        startActivity(intent);
    }
}

File: app/src/main/java/com/ftiuksw/appkantinfti/LoginActivity.kt
package com.ftiuksw.appkantinfti

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.login)
    }

    fun go2Home(view: View?) {
        val intent = Intent(this, HomeActivity::class.java)
        val editText = findViewById<View>(R.id.FirstName_ID) as EditText
        val message = editText.text.toString()
        intent.putExtra("Username", message)
        startActivity(intent)
    }
}